packages/headless/src/app/engine.test.ts

```diff
@@ -90,4 +91,32 @@ describe('engine', () => {
     expect(await engine.renewAccessToken()).toBe('newToken');
     done();
   });
+
+  it("when name is specified in the config, the engine's store name is initialized with the config's value", () => {
+    options.configuration.name = 'myEngine';
+    const spy = jest.spyOn(Store, 'configureStore');
+    initEngine();
+
+    expect(spy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        name: options.configuration.name,
+      })
+    );
+    spy.mockReset();
+    spy.mockRestore();
```

Comment:
Are those two lines really needed? What if you just did:

```
jest.spyOn(Store, 'configureStore');
....
expect(Store.configureStore).toHaveBeenCalledWith(....)
``` 


Also, from the doc https://jestjs.io/docs/mock-function-api#mockfnmockrestore
> Does everything that mockFn.mockReset() does, and also restores the original (non-mocked) implementation.
You'd only need to call mockReset or mockRestore

Reply:
You are right, these lines are unnecessary.